frontend/.gitignore

```diff
@@ -27,3 +27,5 @@ out
 
 # Finder (MacOS) folder config
 .DS_Store
+
+.env.keys
```

Comment:
shouldn't be needed with the new stuff, you should delete the keys from frontend it works with the command from root now.